### ISSUE TITLE
[Error Submission] Error 105-3687

### ISSUE THREAD

**Issue by Antonio225t** [nh-server/Kurisu#1219]
**Console**

Wii U 5.5.5 E

**Error Code**

105-3687

**Error Description**

I started up my Wii U after a long time of not using it, then I put in the disk of **New Super Mario Bros U** and launched **DDD** because I wanted to dump it. After launching DDD I launched the game but to be greated with the message that says that the game needs to be updated, I tried to click exit and reopen the game but the message pops up every time. So I choosed to update the game and the Wii U showed me a message saying that the Wii U needed to be updated (like a FW update). After I did the Wii U update, I launched the game and the updating download process started. It ended after a while successfully and started the installation process, but it didn't even start to install that it showed me **105-3687** error code. From what I can tell it seems like the downloaded files are corrupted or the installer have some issues that I do not know... Thanks for the help in advance 👍,

TL;DR I tried updating NSMBU and in the installation process gave me **105-3687** at the start.

**Comment by Lazr1026** [comment]
Run the Ticket Cleaner app from the Homebrew Appstore.

Also worth noting that this is for error submissions, *not* support.

**Comment by Antonio225t** [comment]
I understood, sorry about the long message about that. Also, 1 hour ago I tried https://github.com/nh-server/Kurisu/issues/920 this and it worked. but I guess it's better the ticked cleaner app. Thanks again
